src/lib/get-feature.js

```diff
@@ -0,0 +1,23 @@
+import buildGeoServerUrl from './build-geoserver-url'
```

Comment:
Isn't there already a getFeature function in the download-helpers? see createWfsParameters